recipe/meta.yaml

```diff
@@ -1,16 +1,18 @@
-{% set version = "0.19.1" %}
+{% set version = "0.19.2" %}
+{% set name = "scikit-learn" %}
 {% set variant = "openblas" %}
 
 package:
-  name: scikit-learn
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
-  sha256: 2947f531ad52e81b2436b26608d8198778ac0b4baa7d2925db8b3b3fcb39c8a2
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b276739a5f863ccacb61999a3067d0895ee291c95502929b2ae56ea1f882e888
 
 build:
-  number: 201
+  number: 0
```

Comment:
So this needs to be `200`.

ref: https://conda-forge.org/docs/blas.html

Reply:
We could also use this trick ( https://github.com/conda-forge/numpy-feedstock/commit/4a98d05bc762321aadb6a385edacb2171d53adad ).

Reply:
Thanks for the review! Set it to 200, I think it's still more explicit that setting then incrementing the `build_number` variable..